app/lib/frontend/templates/views/page/landing_experimental.mustache

```diff
@@ -0,0 +1,60 @@
+{{! Copyright (c) 2020, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+
+{{#has_ff}}
```

Comment:
Can't we just make a condition on `ff_mini_list_html`? and avoid the `has_...` variables?

Reply:
We could, but in the future non-nullability may break such templates unexpectedly, I think it is better to be explicit if the overhead is small.